### PR TITLE
added contributors section

### PR DESCRIPTION
**Pull request by ahmadmunim**
### What are you trying to accomplish?
Adding contributors section

### How are you accomplishing it?
By listing people who've contributed to the wiki along with their github, uwindsor email, and their personal website if they have one

### Is there anything reviewers should know?
nope

### Checks before you create your pull request

-   [X] Did you read and follow article requirements?
-   [X] Did you run prettier?
-   [X] Is it safe to rollback this change if anything goes wrong?

### CODE REVIEW

File: resources/contributors.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
++ wong93@uwindsor.ca
+
+#### **Ahmad Munim**
++ https://github.com/ahmadmunim
++ munima@uwindsor.ca
+
+#### **Arunesh Choudhary**
++ https://github.com/choudh91
+
+#### **Hasan Ahmad**
```
Comment (hassanuahmad) [inline resources/contributors.md]: If you could update my name with "Hassan" (double s)

Comment (ahmadmunim) [reply resources/contributors.md]: Fixed.

**Comment by kilbouri** [comment]
Could you rebase/update this on master please

File: courses/offering/courses.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
             "fall": true,
             "winter": true,
             "summer": true,
+
```
Comment (kilbouri) [inline courses/offering/courses.json]: Is this whitespace necessary?

**Review by kilbouri — APPROVED** [review]
LGTM 🚀 🚀 Just one whitespace-only change that probably can be removed safely.

File: courses/offering/courses.json (hunk 2 above)
Comment (ahmadmunim) [reply courses/offering/courses.json]: Nah it isn't.

**Pull request merged**